README.md

```diff
@@ -0,0 +1,44 @@
+# axl
+
+Python excel add-ins, made easy!
+
+# Requirements
+
+- Microsoft Excel
+- Windows 64 Bit
+- Anaconda or Miniconda
```

Comment:
Might be worthwhile to list `axl`'s package dependencies.

Reply:
Will do.